lib/WebpackOptionsApply.js

```diff
@@ -274,12 +274,12 @@ class WebpackOptionsApply extends OptionsApply {
 		if (!options.experiments.outputModule) {
 			if (options.output.module) {
 				throw new Error(
-					"'output.module: true' is only allowed when 'experiements.outputModule' is enabled"
+					"'output.module: true' is only allowed when 'experiments.outputModule' is enabled"
```

Comment:
aside: Should this be `output.experiments.outputModule` instead?

Reply:
No it's a top-level option